Automattic/vip-go-mu-plugins
# Update default Jetpack version to 10.4

## Description
Update default Jetpack version to 10.4

## Changelog Description

### Plugin Updated: Jetpack 10.4

The default version of Jetpack has been updated to 10.4

## Checklist


## Steps to Test
1. Check out PR.
1. Go to `wp-admin` > `Jetpack`
1. Verify the version of Jetpack is 10.4
